[shopsys] github token erase (#551)
shopsys/shopsys: UPGRADE.md

        - run `docker-sync start` to create volumes
    - run `docker-compose up -d --build --force-recreate` to start application
    - phing target for checking and fixing standards has changed, update `build-dev.xml` according to the changes

> - [#551 - github token erase](https://github.com/shopsys/shopsys/pull/551)

Reviewer: This upgrade note should be optional
```suggestion
- *(optional)* [#551 - github token erase](https://github.com/shopsys/shopsys/pull/551)
```

>     - update all `docker-compose.yml` config files and all dockerfile configs according to the changes

Reviewer: You can be more specific than to force devs to look at the original PR:
```suggestion
    - remove the lines mentioning `github_oauth_token` from your `docker/php-fpm/Dockerfile` and `docker-compose.yml`
```

    - rebuild `php-fpm` container

> ### [shopsys/shopsys]

Reviewer: There should be an optional upgrade note for the monorepo as well:

```markdown
- *(optional)* [#551 - github token erase](https://github.com/shopsys/shopsys/pull/551)
    - you can stop providing the `github_oauth_token` in your `docker-compose.yml`
```
